src/flexalloc_mm.c

```diff
@@ -1398,7 +1398,9 @@ fla_object_read(const struct flexalloc * fs, struct fla_pool const * pool_handle
     sp.strp_nobjs = pool_entry->strp_nobjs;
```

Comment:
Can we also get the w_offset into the strip paramaters? I don't like how we use the strp_nobjs in the s_offset calculation.